Makefile

```
@@ -59,17 +67,30 @@ coverage:
-	@./target/release/gts server --port 8000 & echo $$! > .server.pid
-	@sleep 2
-	@echo "Running e2e tests..."
-	@PYTHONDONTWRITEBYTECODE=1 pytest -p no:cacheprovider --log-file=e2e.log ./.gts-spec/tests || (kill `cat .server.pid` 2>/dev/null; rm -f .server.pid; exit 1)
-	@echo "Stopping server..."
-	@kill `cat .server.pid` 2>/dev/null || true
-	@rm -f .server.pid
-	@echo "E2E tests completed successfully"
+# Create/refresh the Python venv used by the gts-spec e2e test suite
+e2e-venv: $(VENV_PY)
+
+$(VENV_PY):
```

checkmake (0.3.2)

[warning] Target body for "$(VENV_PY)" exceeds allowed length of 5 lines (8).

(maxbodylength)